app/backend/src/couchers/servicers/api.py

```diff
@@ -697,15 +715,17 @@ def SendFriendRequest(self, request, context, session):
             session,
             user_id=friend_relationship.to_user_id,
             topic_action="friend_request:create",
-            key=friend_relationship.from_user_id,
+            key=str(friend_relationship.from_user_id),
```

Comment:
There's currently an implicit conversion there, but we have to be explicit for mypy's happiness